cookbooks/aws-parallelcluster-slurm/kitchen.slurm-config.yml

```diff
@@ -176,3 +176,19 @@ suites:
       cluster:
         node_type: ComputeFleet
         head_node_private_ip: '127.0.0.1'
+  - name: config_head_node_munge
+    run_list:
+    - recipe[aws-parallelcluster-tests::setup]
+    - recipe[aws-parallelcluster-slurm::config_head_node]
+    verifier:
+    controls:
+      - /tag:config_munge/
+    attributes:
+      node_type: HeadNode
+      scheduler: 'slurm'
+      cluster:
+        region: 'us-east-2'
+        config:
+          DevSettings:
+            SlurmSettings:
+              MungeKeySecretArn: 'arn:aws:secretsmanager:us-east-2:249582277112:secret:TestMungeKey-jDtnqi'
```

Comment:
This configuration will only work for this specific account and region. Let's sync on how to proceed.

Reply:
Done!